templates/head.html

```diff
@@ -11,6 +11,13 @@
 {{^includesIframe}}
 <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
 {{/includesIframe}}
+{{^includesGeo}}
+{{^includesConsent}}
+<script async custom-element="amp-geo" src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"></script>    
+<script async custom-element="amp-consent" src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"></script>
+{{/includesConsent}}
+{{/includesGeo}}
+
```

Comment:
Sorry, @sebastianbenz  - when I do it this way, the metadata doesn't get included properly.  To be exact, the <link rel="canonical"> doesn't get added.

Reply:
Are you sure it's ok to nest those template conditionals?

Reply:
Guess not:
```
Message:
    Nesting error: includesGeo vs. includesConsent
Stack:
Error: Nesting error: includesGeo vs. includesConsent
```

Reply:
Um, now it works.  Never mind ;-)